ExcelUtils.js

```diff
@@ -12,7 +12,7 @@ export function createExcel(arrayOfArray, searchName) {
   let name = searchName;
   name = name.split(' ').join('');
 
-  const fileName = `${name}.xlsx`;
+  const fileName = `output/${name}.xlsx`;
```

Comment:
what was the point of this?
It blocks the it blocks the creation of the excel file btw